### PR TITLE
ElementTag.discord_id_to_time tag

### PR DESCRIPTION
**Pull request by davight**
Requested by zemenus in https://discord.com/channels/315163488085475337/1188944433148858468

### CODE REVIEW

File: src/main/java/com/denizenscript/ddiscordbot/properties/DiscordElementTagExtensions.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import net.dv8tion.jda.api.utils.TimeUtil;
+
+public class DiscordElementTagExtensions {
+
+    public static void register() {
+
+        // <--[tag]
+        // @attribute <ElementTag.discord_id_to_time>
+        // @returns TimeTag
+        // @group math
```
Comment (mcmonkey4eva) [inline src/main/java/com/denizenscript/ddiscordbot/properties/DiscordElementTagExtensions.java]: This is missing `// @Plugin dDiscordBot`, and the `group` should be `extensions`

File: src/main/java/com/denizenscript/ddiscordbot/properties/DiscordElementTagExtensions.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        // @description
+        // Each discord ID (for channels, users, guilds, ect) stores the exact time it was created.
```
Comment (mcmonkey4eva) [inline src/main/java/com/denizenscript/ddiscordbot/properties/DiscordElementTagExtensions.java]: `etc.`

Comment (mcmonkey4eva) [reply src/main/java/com/denizenscript/ddiscordbot/properties/DiscordElementTagExtensions.java]: It's not _stored_ - it _is_ a timestamp. ID's are just unix timestamps with some arcane discordmath applied.

**Comment by davight** [comment]
fixed

**Pull request merged**